source/jormungandr/jormungandr/street_network/ride_sharing_journey.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2001-2016, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+
+class RideSharingJourney(object):
+    # https://stackoverflow.com/a/28059785/1614576
+    __slots__ = ('duration',
+                 'distance',
+                 'shape',
+                 'offer_url',
+                 'pickup_addr',
+                 'pickup_lat',
+                 'pickup_lon',
+                 'dropoff_addr',
+                 'dropoff_lat',
+                 'dropoff_lon',
+                 'pickup_date_time',
+                 'dropoff_date_time',
+                 'driver_name',
```

Comment:
A "individual" structure might be nice, to mutualize with potential passengers in the future